helios-client/src/main/java/com/spotify/helios/common/descriptors/RollingOperationTasks.java

```diff
@@ -26,32 +26,32 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Any reason to not have @JsonInclude(JsonInclude.Include.NON_NULL) as we do elsewhere?